common/perf-test-core/src/main/java/com/azure/perf/test/core/PerfStressProgram.java

```diff
@@ -149,6 +160,12 @@ public static void run(Class<?> testClass, PerfStressOptions options) {
                     runTests(tests, options.isSync(), options.getParallel(), options.getDuration(), title);
                 }
             } finally {
+                if (startedPlayback) {
+                    Disposable playbackStatus = printStatus("=== Stop Playback ===", () -> ".", false, false);
+                    Flux.just(tests).flatMap(PerfStressTest::stopPlaybackAsync).blockLast();
```

Comment:
might want to catch exceptions here and resume the program, as any error at this stage will prevent cleanup from happening below.